packages/core/demo/index.scss

```diff
@@ -167,3 +168,79 @@ div.options {
 	background-color: #fff;
 	box-shadow: 0 15px 34px -11px rgba(22, 56, 107, 0.1);
 }
+//**********************************************
+// Themes
+//***********************************************
+
+$gray100: #171717;
+$gray90: #282828;
+$gray10: #f3f3f3;
```

Comment:
are these values not exported in `"~@carbon/themes"`?

Reply:
Theme map only assigns the hex colors to tokens but does not export them as variables. I only added the ones that didnt have tokens assignable within the themes.